tests/nnfw_api/src/one_op_tests/ArgMax.cc

```diff
@@ -40,6 +40,28 @@ TEST_F(GenModelTest, OneOp_ArgMax_AxisToConst)
   SUCCEED();
 }
 
+TEST_F(GenModelTest, OneOp_ArgMax64_AxisToConst)
```

Comment:
Please change name `OneOp_ArgMax64_AxisToConst` to `OneOp_ArgMax_Int64_AxisToConst`

Reply:
Thanks for the comment, I have modified accordingly.